packages/flutter/test/widgets/icon_theme_data_test.dart

```diff
@@ -18,4 +18,13 @@ void main() {
     expect(lerped.opacity, equals(0.625));
     expect(lerped.size, equals(18.0));
   });
+
+  test('IconThemeData lerp with null', () {
```

Comment:
This is good.  Please add a test for IconThemeData.lerp(null, data, 0.25); and IconThemeData.lerp(null, null, 0.25).

Reply:
Good idea, done.